W-11464471-hosting-options (#134)
mulesoft/docs-general: modules/ROOT/pages/intro-platform-hosting.adoc

MuleSoft provides three different options for hosting the Anypoint Platform runtime plane:

* CloudHub 2.0
+

> CloudHub 2.0 is MuleSoft's cloud-based environment for hosting the Mule runtime and related services. CloudHub 2.0 enables you to deploy an API or a Mule application into a container-based platform that's managed by MuleSoft. You can deploy to MuleSoft's shared spaces or your own configurable private spaces for more control over what can interact with your applications.

Reviewer: ```suggestion
CloudHub 2.0 is MuleSoft's new default cloud-hosted environment for hosting the Mule runtime and related services. CloudHub 2.0 enables you to deploy an API or a Mule application into a container-based platform that's managed by MuleSoft. You can deploy to MuleSoft's shared spaces or your own configurable single-tenant private spaces for more control over what can interact with your applications.
```

> CloudHub 2.0 is MuleSoft's new default cloud-hosted environment for hosting the Mule runtime and related services. CloudHub 2.0 enables you to deploy an API or a Mule application into a container-based platform that's managed by MuleSoft. You can deploy to MuleSoft's shared spaces or your own configurable single-tenant private spaces for more control over what can interact with your applications.

Reviewer 2: ```suggestion
CloudHub 2.0 is MuleSoft's default cloud-hosted environment for hosting the Mule runtime and related services. CloudHub 2.0 enables you to deploy an API or a Mule application into a container-based platform that's managed by MuleSoft. You can deploy to MuleSoft's shared spaces or your own configurable single-tenant private spaces for more control over what can interact with your applications.
```

Reviewer 2: it's not always going to be "new"

Reviewer 2: "cloud-hosted" seems weird to me, but leave it if you want

+
CloudHub 2.0 also provides high availability, ingress load balancers, AWS role-based access controls, and endpoint configuration. See xref:cloudhub-2::index.adoc[CloudHub 2.0] for more information.
+

> CloudHub 2.0 can only be managed only by the version of the cloud control plane that exists in the same environment (US cloud or EU cloud).

Author: ```suggestion
CloudHub 2.0 can be managed only by the version of the cloud control plane that exists in the same environment (US cloud or EU cloud).
```

* CloudHub
+

> CloudHub is MuleSoft's original cloud-based environment for hosting the Mule runtime server and related services. CloudHub enables you to deploy an API or a Mule application on a platform that’s managed by MuleSoft.

Reviewer: ```suggestion
CloudHub is MuleSoft's original cloud-hosted environment for hosting the Mule runtime and related services. CloudHub enables you to deploy an API or a Mule application on a VM-based platform that’s managed by MuleSoft.
```

> CloudHub is MuleSoft's original cloud-hosted environment for hosting the Mule runtime and related services. CloudHub enables you to deploy an API or a Mule application on a VM-based platform that’s managed by MuleSoft.

Reviewer 2: ```suggestion
CloudHub is MuleSoft's original cloud environment for hosting the Mule runtime and related services. CloudHub enables you to deploy an API or a Mule application on a VM-based platform that’s managed by MuleSoft.
```

Reviewer 2: I'm bristling at "original" and "new" but can think of a way around it. I def don't think we need to say "new" in docs, tho

+
CloudHub also provides high availability, clustering and failover of your APIs and Mule applications and performs load balancing for them. See xref:runtime-manager::cloudhub.adoc[CloudHub] for more information.
+

> CloudHub can only be managed only by the version of the cloud control plane that exists in the same environment (US cloud, EU cloud, or MuleSoft Government Cloud).

Reviewer: this does not read grammatically pleasing. There are two "only's in one sentence.

Author: ```suggestion
CloudHub can be managed only by the version of the cloud control plane that exists in the same environment (US cloud, EU cloud, or MuleSoft Government Cloud).
```

Not all Anypoint Platform components are supported in each of the Runtime Plane hosting options:

[%header,cols="5*a"]
|===
| Anypoint Platform Component | CloudHub 2.0 | CloudHub | Runtime Fabric | Standalone
| Mule runtime engine | Y | Y | Y | Y
| Anypoint MQ | N | Y | N | N
| Anypoint Object Store | Y | Y | N | N

> | Anypoint DataGraph | Y | Y | N | N

Author: ```suggestion
| Anypoint DataGraph | N | Y | N | N
```
